### PR TITLE
Reorg, add structure for flop and flap

### PR DESCRIPTION
**Pull request by grandizzy**

### CODE REVIEW

File: environment.sh [hunk 1]
```diff
@@ -1,7 +1,9 @@
-SERVER_ETH_RPC_HOST=https://kovan-node-here
+SERVER_ETH_RPC_HOST=https://parity0.kovan.makerfoundation.com
```
Comment (KentonPrescott) [inline environment.sh]: Was this missed in the filter? I'd recommend we keep this endpoint internal..

File: environment.sh [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
 SERVER_ETH_RPC_PORT=8545
-ACCOUNT_ADDRESS=0x40418xxxxxxx
-ACCOUNT_KEY='key_file=/opt/keeper/secrets/keystore.json,pass_file=/opt/keeper/secrets/password.txt'
+ACCOUNT_ADDRESS=0x40418bEB7F24c87ab2d5Ffb8404665414E91D858
```
Comment (KentonPrescott) [inline environment.sh]: Same for this address

File: environment.sh [hunk 3]
```diff
@@ -1,7 +1,9 @@
-SERVER_ETH_RPC_HOST=https://kovan-node-here
+SERVER_ETH_RPC_HOST=https://paritynodehere
 SERVER_ETH_RPC_PORT=8545
-ACCOUNT_ADDRESS=0x40418xxxxxxx
-ACCOUNT_KEY='key_file=/opt/keeper/secrets/keystore.json,pass_file=/opt/keeper/secrets/password.txt'
+ACCOUNT_ADDRESS=0x40418bxxxxxx
```
Comment (KentonPrescott) [inline environment.sh:3]: In an attempt to group keeper types with different environment variables (RPC Host, ETH address, pricing model ETH_URL), wouldn't it make more sense to have an `environment.sh` file associated with flip_eth_a ? Maybe store it in the `flip/eth-a/` directory?

Comment (KentonPrescott) [reply environment.sh:3]: OR could define all environment variables in the same file, e.g.:
`ACCOUNT_FLIP_ETH_A_ADDRESS: ...`
`ACCOUNT_FLIP_BAT_A_ADDRESS: ...`
`ACCOUNT_FLAP_ADDRESS: ... `
` ... `

Comment (grandizzy) [reply environment.sh:3]: Yes, that's my thought: have environment.sh in top dir and configure everything that is needed there (instead one env file for each)

**Pull request merged**